minor editing pass (#220)
scionassociation/scion-cp_I-D: draft-dekater-scion-controlplane.md

~~~

- `segment_info`: This field is used as input for the PCB signature. It is the encoded version of the `SegmentInformation` component which provides basic information about the PCB. This component is specified in detail in [](#seginfo).

> - `as_entries`: Contains the `ASEntry` component of all ASes on the path segment represented by this PCB. The order of the AS entries MUST correspond to the path traversal order.

Reviewer: ```suggestion
- `as_entries`: Contains the `ASEntry` component of all ASes on the path segment represented by this PCB. The order of the AS entries MUST correspond to the path traversal order in the PCB propagation direction.
```
I would clarify this. Also, we use this wording about direction quite a lot.

Author: Done.

The Control Plane RPC APIs rely on QUIC connections over UDP/SCION (see {{I-D.dekater-scion-dataplane}}. Establishing such connection requires the initiator to identify the relevant peer (service resolution) and to select a path to it. Since the Control Service is itself the source of path segment information, the following bootstrapping processes apply:

* Neighboring ASes communicate using one-hop paths, as described in {{I-D.dekater-scion-dataplane}}.

> * Paths to non-neighboring ASes are discovered and constructed incrementally by propagating beacons received from neighbors via these one-hop paths.

Reviewer 2: ```suggestion
* Paths to non-neighboring ASes are constructed from PCBs that are received from neighbors via these one-hop paths.
```
I'm trying to clarify that paths are derived from incoming beacons, not from beacons that the AS propagates (the phrasing seems ambiguous to me).
